.gitignore

```diff
@@ -8,6 +8,7 @@
 
 # testing
 /coverage
+/yarn.lock
```

Comment:
This should never be added to the gitignore since it ensures that CI is able to download exactly the versions of dependencies that the developer was working from

https://classic.yarnpkg.com/en/docs/yarn-lock/

Reply:
Got it - so committing changes to that file, even though they're auto-generated, is fine and should be done blindly?

Reply:
Not exactly, think of yarn.lock as the dependency version of the canary in the coalmine. By pulling in an npm package you could be potentially pulling in 100s of other packages (node packages tend to be very small). If your yarn lock changes it means something underlying your application changed that you may or may not directly depend on.